docs/faq.md

```diff
@@ -343,6 +344,22 @@ See more at [Using Google Container Registry (GCR) with Minikube](https://ryanes
 
 Jib currently requires configuring your build tool to use the appropriate [Java networking properties](https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html) (`https.proxyHost`, `https.proxyPort`, `https.proxyUser`, `https.proxyPassword`).
 
+### What should I do when the registry responds with UNAUTHORIZED?
+
+When the registry returns "401 Unauthorized" and/or `"code":"UNAUTHORIZED"`, most of the time it genuinely means the registry does not see (yet) that you have the right to pull/push images. This is often due to credential misconfiguration. Examples:
+
+* You did not configure auth information in the places where Jib searches for by default.
+   - `$HOME/.docker/config.json`, [one of the configuration files](https://docs.docker.com/engine/reference/commandline/cli/#configuration-files) for the `docker` command line tool. Check the linked document, the [credential store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store) and [credential helper](https://docs.docker.com/engine/reference/commandline/login/#credential-helpers) sections, and [this](https://github.com/GoogleContainerTools/jib/issues/101) for how to configure auth. For example, you can do `docker login` to save auth in `config.json`, but it is often recommended to configure a credential helper (also configurable in `config.json`).
```

Comment:
I'm not sure what "Check the linked document" is referencing. But you can just replace it with "See: "